spec/services/doc_auth/response_spec.rb

```diff
@@ -156,4 +158,31 @@
       expect(response.doc_type_supported?).to eq(true)
     end
   end
+
+  # LG-11942
+  # The following is for the stubbed selfie check value. Replace with
+  # the real tests when selfie implementation gets to that point.
+  describe 'selfie_check_performed?' do
+    before do
+      allow(IdentityConfig.store).
+        to receive(:doc_auth_selfie_capture_enabled).
+        and_return(selfies_enabled)
```

Comment:
Will also need to set sp[:biometric_comparison_required] here to go with the above code change.

Reply:
Same comment as above; I think this is too much for the stub.